qutebrowser/config/configtypes.py

```diff
@@ -1151,6 +1162,17 @@ class UserStyleSheet(File):
     def __init__(self):
         super().__init__(none_ok=True)
 
+    def transform(self, value):
+        if not value:
+            return None
+        path = super().transform(value)
+        path = os.path.expandvars(path)
+        if os.path.isabs(path):
```

Comment:
Testing with ```isfile``` or ```exists``` results in two failed tests
```
FAIL TestUserStyleSheet::()::test_transform_file
FAIL TestUserStyleSheet::()::test_transform_file_expandvars
```
If I interpret that right the first one fails because the file **file:///foo/bar** doesn't actually exist and we thus reach the else. In that case the test probably needs to change, but I'm not sure how. Should it create a file first and then expect to find it?

Same goes for the second failed test, I think.